src/cosmology/compat/camb/flrw.py

```diff
@@ -0,0 +1,416 @@
+
+    def efunc(self, z: NDFloating, /) -> NDFloating:
+        """Standardised Hubble function :math:`E(z) = H(z)/H_0`."""
+        return self.H(z) / self.H0
+
+    def inv_efunc(self, z: NDFloating, /) -> NDFloating:
+        """Inverse of ``efunc``."""
+        return self.H0 / self.H(z)
+
+    # ----------------------------------------------
+    # Omega
+
+    def Otot(self, z: NDFloating, /) -> NDFloating:
+        r"""Redshift-dependent total density parameter.
+
+        This is the sum of the matter, radiation, neutrino, dark energy, and
+        curvature density parameters.
+
+        .. math::
+
+            \Omega_{\rm tot} = \Omega_{\rm m} + \Omega_{\rm \gamma} +
+            \Omega_{\rm \nu} + \Omega_{\rm de} + \Omega_{\rm k}
+        """
+        return self.cosmo.get_background_densities(
+            self.scale_factor(z),
+            vars=["total"],
+            format="array",
+        )[:, 0]
+
+    def Om(self, z: NDFloating, /) -> NDFloating:
+        """Redshift-dependent non-relativistic matter density parameter.
+
+        Notes
+        -----
+        This does not include neutrinos, even if non-relativistic at the
+        redshift of interest; see `Onu`.
+        """
+        return np.asarray(
+            self.cosmo.get_Omega("cdm", z) + self.cosmo.get_Omega("baryon", z),
+        )
+
+    def Ob(self, z: NDFloating, /) -> NDFloating:
+        """Redshift-dependent baryon density parameter.
+
+        Raises
+        ------
+        ValueError
+            If ``Ob0`` is `None`.
+        """
+        return np.asarray(self.cosmo.get_Omega("baryon", z))
+
+    def Odm(self, z: NDFloating, /) -> NDFloating:
+        """Redshift-dependent dark matter density parameter.
+
+        Raises
+        ------
+        ValueError
+            If ``Ob0`` is `None`.
+
+        Notes
+        -----
+        This does not include neutrinos, even if non-relativistic at the
+        redshift of interest.
+        """
+        return np.asarray(self.cosmo.get_Omega("cdm", z))
+
+    def Ok(self, z: NDFloating, /) -> NDFloating:
+        """Redshift-dependent curvature density parameter."""
+        return np.asarray(self.cosmo.get_Omega("K", z))
+
+    def Ode(self, z: NDFloating, /) -> NDFloating:
+        """Redshift-dependent dark energy density parameter."""
+        ...
+
+    def Ogamma(self, z: NDFloating, /) -> NDFloating:
+        """Redshift-dependent photon density parameter."""
+        ...
+
+    def Onu(self, z: NDFloating, /) -> NDFloating:
+        r"""Redshift-dependent neutrino density parameter.
+
+        The energy density of neutrinos relative to the critical density at each
+        redshift. Note that this includes their kinetic energy (if
+        they have mass), so it is not equal to the commonly used :math:`\sum
+        \frac{m_{\nu}}{94 eV}`, which does not include kinetic energy.
+        Returns `float` if the input is scalar.
+        """
+        ...
```

Comment:
One or a combination of `self.cosmo.get_Omega("neutrino", z)` (massless) and `self.cosmo.get_Omega("nu", z))` (massive)?